```
Changes to docs detected, versions 4.1.yaml 4.2.yaml 4.3.yaml 
if find docs/ -type f -name '*.md' | xargs grep -E '\s+$'; then \
	echo "trailing whitespace found in docs/ (see above)"; \
	echo "run 'make docs-fix-whitespace' to fix it"; \
	exit 1; \
fi
docs/testplan.md:- [ ] Verify that prometheus metrics are accurate. 
trailing whitespace found in docs/ (see above)
run 'make docs-fix-whitespace' to fix it
make: *** [Makefile:215: docs-test-whitespace] Error 1
```

For some reason `test-docs` didn't run on #4043 where some trailing space was introduced, so now the tests are failing.

--- a/docs/testplan.md
+++ b/docs/testplan.md
@@ -366,7 +366,7 @@ tsh bench -i --duration=4h --threads=10 user@teleport-monster-6757d7b487-x226b p
 
 Observe prometheus metrics for goroutines, open files, RAM, CPU, Timers and make sure there are no leaks
 
-- [ ] Verify that prometheus metrics are accurate. 
+- [ ] Verify that prometheus metrics are accurate.
 
 **Breaking load tests**
 

